### PR TITLE
Add API Testing with saucectl doc

### PR DESCRIPTION
**Pull request by FriggaHel**
### Description

Adds documentation for API Testing through `saucectl` (and removes `apifctl`.

### Motivation and Context

Uniformization.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing/integrations/apitesting-saucectl-integration.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+Navigate to the root of the API Testing project you just cloned, then use the `run` command to execute the sample test included with the `saucectl` example.
+
+```
+cd saucectl-apitest-example
+saucectl run
+```
+
+The console displays the executing tests, distinguishing which mode is running.
+
+The results and test assets are available immediately following test completion in your [Sauce Labs account](https://app.saucelabs.com/dashboard/tests/vdc).
```
Comment (aliciakrebs) [inline docs/api-testing/integrations/apitesting-saucectl-integration.md:101]: The results are available in  APIT... 
```suggestion
The results and test assets are available immediately following test completion in your [Sauce Labs account](https://app.saucelabs.com/dashboard/tests/vdc).
```

File: docs/api-testing/integrations/yaml.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
+---
+id: yaml
+title: Configuring saucectl for API Testing
+sidebar_label: YAML Configuration
```
Comment (aliciakrebs) [inline docs/api-testing/integrations/yaml.md:4]:
```suggestion
sidebar_label: saucectl YAML Configuration
```

**Review by LoryApiFortress — APPROVED** [review]

File: docs/api-testing/integrations/apitesting-saucectl-integration.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+1. Run the `configure` command:
+
+   ```
+   saucectl configure
+   ```
+
+1. Enter your Sauce Labs Username and Access Key at the prompts.
+
+
+## Step 4: Clone the API Testing Project
```
Comment (aliciakrebs) [inline docs/api-testing/integrations/apitesting-saucectl-integration.md:64]:
```suggestion
## Step 4: Clone the API Testing GitHub Project
```

File: docs/api-testing/integrations/apitesting-saucectl-integration.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+git clone git@github.com:saucelabs/saucectl-apitest-example.git
+```
+
+</TabItem>
+</Tabs>
+
+
+## Step 5: Run Tests
+
+Navigate to the root of the API Testing project you just cloned, then use the `run` command to execute the sample test included with the `saucectl` example.
```
Comment (aliciakrebs) [inline docs/api-testing/integrations/apitesting-saucectl-integration.md:92]:
```suggestion
Navigate to the root of the API Testing GitHub project you just cloned, then use the `run` command to execute the sample test included with the `saucectl` example.
```

File: docs/api-testing/integrations/apitesting-saucectl-integration.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+import TabItem from '@theme/TabItem';
+
+Using the [`saucectl` CLI](/dev/cli/saucectl), you can execute API tests and interact with Sauce Labs API Testing.
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+- Your Sauce Labs [Username and Access Key](https://app.saucelabs.com/user-settings).
+- An existing API Testing Project. For details on how to create one, see [API Testing Quickstart](/api-testing/quickstart/).
+
```
Comment (aliciakrebs) [inline docs/api-testing/integrations/apitesting-saucectl-integration.md:19]: - Have [Git](https://git-scm.com/downloads) and [GitHub](https://docs.github.com/) set up.

**Review by aliciakrebs — APPROVED** [review]

File: docs/api-testing/integrations/apitesting-saucectl-integration.md (hunk 5 above)
Comment (FriggaHel) [reply docs/api-testing/integrations/apitesting-saucectl-integration.md:19]: That's something we never mentioned on other projets. Should we add this ?
Knowing that a Github account is not even needed, and we expect a user to know how to use git.

Comment (aliciakrebs) [reply docs/api-testing/integrations/apitesting-saucectl-integration.md:19]: thought it would  be needed to work with the example repo.

**Pull request merged**